inst/rmarkdown/templates/reprex_document/resources/preview.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+
+<html xmlns="http://www.w3.org/1999/xhtml"$if(lang)$ lang="$lang$" xml:lang="$lang$"$endif$>
+
+<head>
+
+<meta charset="utf-8">
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+<meta name="generator" content="pandoc" />
+<meta name="viewport" content="width=device-width, initial-scale=1">
+
+<link rel="stylesheet" href="$github-markdown-css$">
+<style>
+body {
+  box-sizing: border-box;
+  min-width: 200px;
+  max-width: 980px;
+  margin: 0 auto;
+  padding: 45px;
+  padding-top: 15px;
+}
+</style>
+
+$if(math)$
+  $math$
+$endif$
+
+</head>
+
+<body>
+
+<div class="markdown-body">
```

Comment:
New div because `markdown-body` that is the main class that GitHub's CSS deals with.